SPECS/sriov-network-device-plugin/sriov-network-device-plugin.spec

```diff
@@ -0,0 +1,35 @@
+Summary:        Plugin for discovering and advertising networking resources
+Name:           sriov-network-device-plugin
+Version:        3.3
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin
+Source0:        https://github.com/k8snetworkplumbingwg/%{name}/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+BuildRequires:  golang
+
+%description
+sriov-network-device-plugin is Kubernetes device plugin for discovering and advertising networking
+resources in the form of SR-IOV virtual functions and PCI physical functions
+
+%prep
+%autosetup -p1
+
+%build
+go build -mod vendor -o ./build/sriovdp ./cmd/sriovdp/
+
+%install
+install -D -m0755 build/sriovdp %{buildroot}%{_bindir}/sriovdp
+install -D -m0755 images/entrypoint.sh %{buildroot}%{_bindir}/entrypoint.sh
+
+%files
```

Comment:
From the [upstream docker](https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/blob/master/images/Dockerfile), looks like ddptool is also needed. Are you planning to include it in the spec here or when building the docker file?

Looks like upstream they checked-in directly under [images](https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/tree/master/images)

Reply:
I plan on including it when building the docker file. What are your thoughts?

Reply:
I would suggest including it here like upstream so the version of ddptools is aligned with the built sriov-dp.

Reply:
I have added in the ddptool file in the spec, and it is included in the built rpm file - this conversation can now be resolved